### PR TITLE
Update theme watch to include js & images.

### PR DESCRIPTION
**Pull request by dipakmdhrm**
#324 

- Updated theme watch to include js & images.
- Updated README

### CODE REVIEW

**Review by AronNovak — APPROVED** [review]
It makes sense, thanks! Indeed it's a frustration when custom code doesn't get styling due to the fact that we don't watch that.

The only thing I'd advise is to not introduce redundancy for documentation. On the long run one of the `README.md`s will become outdated. Let's keep the existing README file, and if you prefer, let's just refer to the top README from the theme README, instead of duplicating part of its content.

**Comment by dipakmdhrm** [comment]
> It makes sense, thanks! Indeed it's a frustration when custom code doesn't get styling due to the fact that we don't watch that.
> 
> The only thing I'd advise is to not introduce redundancy for documentation. On the long run one of the `README.md`s will become outdated. Let's keep the existing README file, and if you prefer, let's just refer to the top README from the theme README, instead of duplicating part of its content.

Done.

File: web/themes/custom/server_theme/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Theme Development
 
+To learn about theme directory structure and commands to compile and watch
```
Comment (amitaibu) [inline web/themes/custom/server_theme/README.md:3]: Let's remove this file - and move all info to the main README.md

Comment (dipakmdhrm) [reply web/themes/custom/server_theme/README.md]: Done.

**Pull request merged**